### PR TITLE
Add output alerts

### PR DESCRIPTION
**Pull request by KevinJoiner**

### CODE REVIEW

File: tests/prom/alerts-modified.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+
+groups:
+  - name: dis-service
+    rules:
+      - alert: NoDataReceived
+        expr: sum(rate(processor_sent{namespace="dev", container="dis", label="convert_cloudevent"}[30m])) BY (pod, job, namespace) == 0
```
Comment (zakharenkodmytro) [inline tests/prom/alerts-modified.yaml:6]: probably should not be there under benthos folder

Comment (KevinJoiner) [reply tests/prom/alerts-modified.yaml:6]: it's under the prom folder

Comment (KevinJoiner) [reply tests/prom/alerts-modified.yaml:6]: but I want to ignore this file

Comment (zakharenkodmytro) [reply tests/prom/alerts-modified.yaml:6]: Probably you do commits so quick)

File: tests/prom/rules-tests.yaml [hunk 2]
```diff
@@ -54,3 +54,36 @@ tests:
...
+              namespace: dev
+            exp_annotations:
+              summary: "No data sent to S3 in the last 30 minutes"
+              description: "Please check dis service and networking."    
+
+  # Test for NoDataSentClickhouse alert
+  - interval: 6m
+    input_series:
+      - series: 'output_batch_sent{namespace="dev", container="dis", label="insert_signal_clickhouse"}'
+        values: '35 35 35 35 35 35 35'  # 30m of no data
```
Comment (zakharenkodmytro) [inline tests/prom/rules-tests.yaml:78]: nit, you have only 6 datapoints but the eval_time is happening on 10th datapoint, I think missed datapoints treated as 0. You can just add more datapoints to cover all 60 mins

**Review by zakharenkodmytro — APPROVED** [review]

**Pull request merged**